.travis.yml

```diff
@@ -23,11 +24,16 @@ matrix:
     - python: 3.6
       env: TOXENV=flake8
 
+before_install:
```

Comment:
There another syntax that doesn't require sudo and runs a lot faster, (basically, just use `apt: DBus`.